src/lib/pipeline_configuration.py

```diff
@@ -745,21 +749,13 @@ def validate(self):
         validators.validate_bool(self.filter_test_messages, "filter_test_messages")
         validators.validate_bool(self.move_ws_messages, "move_ws_messages")
 
-        if self.pipeline_name is not None:
-            validators.validate_string(self.pipeline_name, "pipeline_name")
-
         if self.drive_upload is not None:
             assert isinstance(self.drive_upload, DriveUpload), \
                 "drive_upload is not of type DriveUpload"
             self.drive_upload.validate()
 
-        validators.validate_string(self.memory_profile_upload_url_prefix, "memory_profile_upload_url_prefix")
-
-        if self.listening_group_csv_urls is not None:
-            validators.validate_list(self.listening_group_csv_urls, "listening_group_csv_urls")
-            for i, listening_group_csv_url in enumerate(self.listening_group_csv_urls):
-                validators.validate_string(listening_group_csv_url, f"{listening_group_csv_url}")
-
+        memory_profile_upload_url_prefix = f"{self.memory_profile_upload_bucket}{self.bucket_dir_path}"
+        validators.validate_string(memory_profile_upload_url_prefix, "memory_profile_upload_url_prefix")
```

Comment:
Better to validate_url with the gs test, and to validate all of the new variables

Reply:
updated thanks.